easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-foss-2021b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'GlobalArrays'
+version = '5.8'
+
+homepage = 'https://hpc.pnl.gov/globalarrays'
+description = "Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/GlobalArrays/ga/releases/download/']
+sources = ['v%(version)s/ga-%(version)s.tar.gz']
+checksums = ['64df7d1ea4053d24d84ca361e67a6f51c7b17ed7d626cb18a9fbc759f4a078ac']
+
+dependencies = [('imkl', '2021.4.0', '', SYSTEM)]
+
+configopts = ' --with-mpi --enable-i8'
+configopts += ' --with-blas8="-L$MLROOT/lib/intel64 -lmkl_sequential -lmkl_intel_ilp64"'
+configopts += ' --with-scalapack8="L$MKLROOT/lib/intel64 -lmkl_scalapack_ilp64 -lmkl_intel_ilp64 '
+configopts += '-lmkl_sequential -lmkl_core -lmkl_blacs_intelmpi_ilp64 -lpthread -lm -ldl"'
```

Comment:
`intelmpi`?

Reply:
Thanks for finding that! I am surprised that that all worked to be honest. 
I have downgraded to `gompi` do any instances of `OpenBLAS` and `SCALARPAC` are coming from `MKL` only. 
That is compiling but 4 test jobs are failing and hence I leave that as WIP.
That said, in PR #16255 I am using `gompi-2021a` and here all test jobs are passed. If anybody has a good idea why they are not working here, please let me know. I would like to use this version for the `OpenMolcas` installation I need where the testjobs are failing using the Intel compiler.